app/models/comment.rb

```diff
@@ -1,2 +1,9 @@
 class Comment < ActiveRecord::Base
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
```

Comment:
- [ ] OPTIONAL
It's redundant to include `class_name: 'Post'`  to _comment_ association with _post_. Simply do something like:
```ruby
# app/model/comment.rb
belongs_to :post